### PR TITLE
#1781 - DataGuideDialog content update

### PR DESCRIPTION
**Pull request by maciekz1996**
#1781 

**Changes**
* Updated the content of SearchGuide in Data Portal
* Renamed SearchGuideDialog component to DataGuideDialog

**Screenshots**
Screen Shot 2021-05-05 at 13 13 07

### CODE REVIEW

**Review by sergioloporto — APPROVED** [review]
all good

**Comment by abhidg** [comment]
Thanks! I hadn't reviewed the text before the PR - but we aren't using keywords anymore right? I would reword the first sentence of the last paragraph to: "For full-text search, enter any combination of search terms." I'm not sure what the full text search is doing at the moment (#1529 is still open). @joe-brilliant @Mougk thoughts?

**Comment by joe-brilliant** [comment]
Sounds great to me @abhidg !

**Comment by maciekz1996** [comment]
@abhidg @joe-brilliant Full-test search is currently searching through cases in mongodb based on `text indexes` on several fields in case schema. I've just checked on https://dev-curator.ghdsi.org/cases?q=Turkey and indeed all the results are connected to Turkey. For example other cases that are retrieved from this search query which are not from country Turkey contain Turkey in some fields in case details like for example `Travelled from Turkey` etc.

So in my opinion we can close #1529 as the results are correct, both on production and dev. 

I will also replace the content of data guide as @abhidg suggested

**Review by abhidg — CHANGES_REQUESTED** [review]
Looks good, just the full-text search para needs updating

**Comment by maciekz1996** [comment]
@abhidg I've updated data guide content
